www/source/blog/2018-10-12-shelving-composites.html.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Shelving Composites (for now)
+date: 2018-10-12
+author: Garrett Amini
+tags: supervisor, packaging
+category: supervisor
+classes: body-article
+---
+
+Nearly exactly one year ago today, Christopher Maier [presented the ability to create _composite packages_](https://www.habitat.sh/blog/2017/10/Introducing-Composites/), a special kind of Habitat package that includes other Habitat packages in order to be managed by a single Supervisor. The Habitat community had been finding all sorts of creative ways to use Habitat to incorporate their services under a single Supervisor, so we sought a way to canonically enable them to do so. 
+
+This past year has taught us that while we were on the right track with the idea of composite packages, our implementation left much to be desired. Users kept trying to utilize composite packages, but would inevitably find themselves bumping up against limitations of the feature, and would thus end implementing the same sorts of creative solutions we were trying to simplify in the first place.
+
+## What's Happening Now?
+
+We will soon be removing the ability to run composite plans from Habitat. In the short term, this will simplify our roadmap to 1.0 and increase critical code stability. We have learned much over the past year from our initial implementation, and based upon that feedback and with an eye toward hearing more from the community, we're looking forward to being able to build new features to accomplish the same things that composite plans were designed to do. 
+
+## The Path Forward
+
+As we near 1.0, we are seeking to make Habitat as robust and reliable as possible. Once we achieve that milestone, we're looking forward to once again providing ways of efficiently limiting binds to the same supervisor.

```

Comment:
This seems a bit redundant with the above paragraph. I'd also leave out mention of any potential concrete solutions like "limiting binds to the same supervisor"; keeping it to "meeting the usecases that composites were designed for" is good enough.